README.md

```diff
@@ -101,15 +100,15 @@ The following tables lists the configurable parameters of the chart and their de
 | allowExternalDNS.domainFilter                                                      | aws route 53 hosted zonezone                                                                                                             | value from aws route53 list-hosted-zones --query "HostedZones[*].Name"                                                                         |
 | allowExternalDNS.txtOwnerID                                                        | identifies externalDNS instance                                                                                                          |                                                                                                                                                |
 | allowExternalDNS.serviceAccount.annotations.eks.amazonaws.com/role-arn             | assuming 'enableIRSA:true' pass the role arn for the external dns                                                                        |                                                                                                                                                |
-| allowEFS.enabled                                                                   | Enables creation of EFS manifests                                                                                                        | FALSE                                                                                                                                          |
-| allowEFS.efs                                                                       | efs file system url                                                                                                                      | FALSE                                                                                                                                          |
-| allowEFS.orgName                                                                   | organization name to configure efs path                                                                                                  | /                                                                                                                                              |
+| allowNFS.enabled                                                                   | Enables creation of EFS manifests                                                                                                        | FALSE                                                                                                                                          |
+| allowNFS.server                                                                    | efs file system url                                                                                                                      | FALSE                                                                                                                                          |
```

Comment:
Mention in the Readme that we can use `allowNFS` for both AWS EFS or other NFS setups as long as they use NFS v4.